docs/submitters/submissionGuide.md

```diff
@@ -0,0 +1,169 @@
+# Submission Guide
+
+If an add-on author wishes to submit their add-on to be visible in this add-on store they will need to:
+- Copy and fill out the metadata template
+- Create a pull request to merge the branch to master of this repository.
+
+## Pre-requisites:
+- Familiarity with GitHub
+- Familiarity with Git, including working with branches.
+
+## Set up
+Fork the `addon-datastore` repository
+
+## Create an add-on version file for submission
+You can create this manually, or generate this from an add-on manifest file.
+
+### Manual file creation
+On a new branch, copy the `_template_addon_release.json` file. 
+- Rename and move the file to `addons/<addonID>/<version>.json`
+- `<addonID>` is the ID of the add-on. This should match the `name` field in the add-on manifest, e.g. "speechPlayer"
+- `<version>` is the add-on version in the form: `Major.Minor.Patch` e.g. "2.4.1"
+
+### Generating from an add-on manifest
+1. Install requirements to python environment:
+`pip install requests configobj`
+1. Convert an add-on to a json file:
+`python convertAddonManifest.py [addonId] [stable|beta|dev] [addonUrl]`
+	- `addonID` is the ID of the add-on. This should match the `name` field in the add-on manifest, e.g. "speechPlayer"
+	- `stable|beta|dev` are the channel options, choose one of these.
+	- `addonUrl` is where the addon file is hosted.
+1. Edit the auto-generated json file as required.
+
+## Filling out the template
+
+### addonId
+The ID for the addon.
+This should match the name field in the addon manifest and the folder name for the submission.
+The convention is a camelCaseName.
+
+Example: `easyAddonTech`
+
```

Comment:
I think we want to encourage migration away from legacy add-on IDs where possible.
This would require updating dependencies such as the translation system registration and ikiwiki